examples/metalk8s-solution-example/example-operator/deploy/crds/solution_v1alpha1_example_cr.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: solution.com/v1alpha1
+apiVersion: scality.com/v1alpha1
```

Comment:
No, that's not a suitable `apiVersion`. Should be `example-solution.metalk8s.scality.com/v1alpha1` or something along those lines (applies everywhere else as well).